From: NIIBE Yutaka <gniibe@fsij.org>
Date: Mon, 17 Apr 2017 09:08:31 +0900
Subject: g10: Fix import/export filter property match.

* g10/import.c (impex_filter_getval): Fix to "else if".

Signed-off-by: NIIBE Yutaka <gniibe@fsij.org>
(cherry picked from commit af5f8ecf51f5e1f33e832d4946d02313b78a0536)
---
 g10/import.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/g10/import.c b/g10/import.c
index 4e6f692..125b994 100644
--- a/g10/import.c
+++ b/g10/import.c
@@ -1235,7 +1235,7 @@ impex_filter_getval (void *cookie, const char *propname)
           snprintf (numbuf, sizeof numbuf, "%d", pk->pubkey_algo);
           result = numbuf;
         }
-      if (!strcmp (propname, "key_created"))
+      else if (!strcmp (propname, "key_created"))
         {
           snprintf (numbuf, sizeof numbuf, "%lu", (ulong)pk->timestamp);
           result = numbuf;
